<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceWnd</class>
 <widget class="QDialog" name="PreferenceWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="EditRaw">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit
Raw Json</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>240</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Warning: Some of the changes
 may work only if you
restart the program.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="coreCombo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>90</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>RenderCore:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Second to Move When
Pressing &quot;Forward&quot;(&quot;Back&quot;):</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="mTimeSpin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.200000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Midi Device:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceWnd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>0</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceWnd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
